src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml

```diff
@@ -185,7 +185,6 @@
           Margin="0,12"
           Content="{x:Static nuget:Resources.Button_IAccept}"
           AutomationProperties.AutomationId="AcceptButton"
```

Comment:
in addition to the fix by removing the labeledby property setting...
seems like the _licenseText element should also remove the AutomationProperties.Name that is set to {Binding Text}.
First of all, it was trying to bind to the text property, i think. But TextBlock.Text already is the default automationproperties.name, so it was unnecessary.  It also probably didn't work the way the author wanted to.